src/lib/pages/LoginConfigTotp.tsx

```diff
@@ -29,7 +29,7 @@ export default function LoginConfigTotp(props: PageProps<Extract<KcContextBase,
 
                             <ul id="kc-totp-supported-apps">
                                 {totp.policy.supportedApplications.map(app => (
-                                    <li>{app}</li>
+                                    <li>{ msg(app as MessageKeyBase) == undefined? app : msg(app as MessageKeyBase) }</li>
```

Comment:
```suggestion
                                    <li>{msg(app as MessageKeyBase)}</li>
```
No need, it will fallback automatically

Reply:
Oh damn very sorry about this, I think it was fixed latter on... I'll adress this hold on.

Reply:
Should be fixed on the latest v6 that is beeing published. Sorry for the messy review

Reply:
No worries, took the latest, all good!